tests/Unit/AuthModuleTest.php

```diff
@@ -116,6 +119,17 @@ public function testAuth() {
 		$user = $this->authModule->auth($request);
 		$this->assertNotNull($user);
 		$this->assertEquals($this->userId, $user->getUID());
+
+		// Valid request with LongUserID
+		$request = $this->getMockBuilder(IRequest::class)->getMock();
+		$this->accessToken->setUserId($this->userIdConcat);
+		$request->expects($this->once())
+			->method('getHeader')
+			->with($this->equalTo('Authorization'))
+			->will($this->returnValue('Bearer ' . $this->accessToken->getToken()));
+		$user = $this->authModule->auth($request);
```

Comment:
The userManager mock statically returns one and the same user all the time. IMO we should not check if we get the correct user here, but instead if `IUserManager::get` is called with the short user id correctly.

Reply:
Oh, actually it doesn't. Then I don't get, yet, why this test is succeeding, because the `$userIdConcat` has the wrong order of params.

Reply:
ok. This is no real Unit test. I really writes users to the DB
Corrected the scenario.